packages/core/admin/shared/contracts/user.ts

```diff
@@ -0,0 +1,46 @@
+import { errors } from '@strapi/utils';
+
+import type { SanitizedAdminUser } from './shared';
+import type { Entity, EntityService } from '@strapi/types';
+
+/**
+ * /find - Find admin users
+ */
+export declare namespace FindAll {
+  // TODO make the types for this
+  export interface Request {
+    query: EntityService.Params.Pick<'admin::user', 'sort' | 'filters' | 'fields'>;
+    body: {};
+  }
+
+  export interface Response {
+    data: {
+      results: SanitizedAdminUser[];
+      pagination: {
+        page: number;
+        pageSize: number;
+        pageCount: number;
+        total: number;
+      };
```

Comment:
Should we make this a more abstract type, because most endpoints are going to need it?

Reply:
I think when we need it we can, I haven't looked at reconciling any of these contracts yet..